backend/app/views/spree/admin/cancellations/index.html.erb

```diff
@@ -34,7 +34,15 @@
             <% end %>
           </td>
           <td class="inventory-unit-quantity align-center">1</td>
-          <td class="inventory-unit-state align-center"><%= inventory_unit.state %></td>
+          <td class="inventory-unit-state align-center">
+            <span class="pill pill-<%= inventory_unit.state %>">
+              <%= Spree.t(
+                inventory_unit.state,
+                scope: :inventory_states,
+                default: inventory_unit.state.titleize
+              ).titleize %>
```

Comment:
👍 will make changes to the translations and then `downcase` or somehow slugify for the class names.